apps/demos/Demos/Stepper/Overview/jQuery/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+  <head>
+    <title>DevExtreme Demo</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=5.0" />
+    <script src="../../../../node_modules/jquery/dist/jquery.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="../../../../node_modules/devextreme-dist/css/dx.light.css" />
+    <script src="../../../../node_modules/devextreme-dist/js/dx.all.js"></script>
+    <script src="data.js"></script>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <script src="index.js"></script>
+  </head>
+  <body class="dx-viewport">
+    <div class="demo-container">
+      <div class="dx-viewport">
+        <div class="demo-container">
+          <div class="stepper-demo">
+            <div class="widget-container">
+              <div class="widget-wrapper widget-wrapper-horizontal">
+                <div class="stepper-wrapper">
+                  <div class="stepper-label">Icons with Titles</div>
+                  <div id="withText"></div>
+                </div>
+                <div class="stepper-wrapper">
+                  <div class="stepper-label">Numbers with Titles</div>
+                  <div id="withIconAndText"> </div>
+                </div>
+                <div class="stepper-wrapper">
+                  <div class="stepper-label">Custom Text</div>
+                  <div id="withIcon"></div>
+                </div>
+              </div>
+            </div>
+
+            <div class="options">
+              <div class="caption">Options</div>
+
+              <div class="option">
+                <div>Orientation</div>
+                <div id="orientation"></div>
+              </div>
+              <div class="option">
+                <div>Navigation Mode</div>
+                <div id="navigation-mode"></div>
```

Comment:
I would prefer to use 'novigationMode' notation instead for id attr value, because you use it for steppers already